src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -239,17 +239,17 @@ private void setLookAndFeel() {
             if (Globals.prefs.getBoolean(JabRefPreferences.USE_DEFAULT_LOOK_AND_FEEL)) {
                 // FIXME: Problems with GTK L&F on Linux and Mac. Needs reevaluation for Java9
                 if (GTK_LF_CLASSNAME.equals(systemLookFeel)) {
-                    // Metal L&F
                     lookFeel = NIMBUS_LOOK_AND_FEEL;
-                    LOGGER.warn("There seem to be problems with OGTK Look&Feel. Using Nimbus L&F instead. Change to another L&F with caution.");
+                    LOGGER.warn("There seems to be problems with GTK Look&Feel. Using Nimbus L&F instead. Change to another L&F with caution.");
                 } else {
                     lookFeel = systemLookFeel;
                 }
             } else {
                 lookFeel = Globals.prefs.get(JabRefPreferences.WIN_LOOK_AND_FEEL);
             }
 
-            if (UIManager.getCrossPlatformLookAndFeelClassName().equals(lookFeel) && !GTK_LF_CLASSNAME.equals(lookFeel)) {
+            //Prevent metal l&f
```

Comment:
use Default L&F:
1. System L&F = gtk => Nimbus
2. System L&F = win => win
3. System L&F = metal look and feel 

Check cross platform 
1. Metal look and feel => Nimbus